src/frontends/tensorflow/src/op/concat.cpp

```diff
@@ -15,36 +15,52 @@ namespace ov {
 namespace frontend {
 namespace tensorflow {
 namespace op {
+OutputVector translate_concat_base(const NodeContext& node, const OutputVector& inputs, int64_t axis) {
```

Comment:
This function is used only once and contains just 2 lines of code which doesn't seem like very useful, is it really needed to have it?

Reply:
Fixed